.github/workflows/motion-benchmarks.yml

```diff
@@ -59,18 +59,29 @@ jobs:
         sudo -Hu testbot bash -lc "go test ./... -v -run TestScores --baselineDir=$BASELINE --modifiedDir=$MODIFIED"
         cat results/motion-benchmarks.md
 
-    # Now that RDK is public, can't directly comment without token having full read/write access
-    # motion-benchmarks-comment.yml will trigger seperately and post the actual comments
-
-    - name: Prepare code comment
-      run: |
-        echo "PR_NUMBER=${{ github.event.pull_request.number }}" >> pr.env
-
-    - name: Upload results
+    - name: Upload results for comment
       uses: actions/upload-artifact@v4
       with:
-        name: pr-motion-benchmark
-        path: |
-          pr.env
-          motion-testing/results/motion-benchmarks.md
+        name: motion-benchmarks-results
+        path: motion-testing/results/motion-benchmarks.md
         retention-days: 1
+
+
+  motion_comment:
```

Comment:
can probably do these steps on the previous job rather than defining a new job, but if this works no objection

Reply:
the only reason i'm putting it here is because. it seems like we may run the benchmark job on a branch directly and not on a PR. At least by the way the job is setup.